Sources/Extensions/CG+Hero.swift

```diff
@@ -40,107 +40,109 @@ internal struct KeySet<Key: Hashable, Value: Hashable> {
 }
 
 internal extension CGSize {
-  internal var center: CGPoint {
+  var center: CGPoint {
     return CGPoint(x: width / 2, y: height / 2)
   }
-  internal var point: CGPoint {
+  var point: CGPoint {
     return CGPoint(x: width, y: height)
   }
-  internal func transform(_ t: CGAffineTransform) -> CGSize {
+  func transform(_ t: CGAffineTransform) -> CGSize {
     return self.applying(t)
   }
-  internal func transform(_ t: CATransform3D) -> CGSize {
+  func transform(_ t: CATransform3D) -> CGSize {
     return self.applying(CATransform3DGetAffineTransform(t))
   }
 }
 
 internal extension CGRect {
-  internal var center: CGPoint {
+  var center: CGPoint {
     return CGPoint(x: origin.x + width / 2, y: origin.y + height / 2)
   }
-  internal var bounds: CGRect {
+  var bounds: CGRect {
     return CGRect(origin: CGPoint.zero, size: size)
   }
   init(center: CGPoint, size: CGSize) {
     self.init(x: center.x - size.width / 2, y: center.y - size.height / 2, width: size.width, height: size.height)
   }
 }
 
-extension CGFloat {
-  internal func clamp(_ a: CGFloat, _ b: CGFloat) -> CGFloat {
+internal extension CGFloat {
+  func clamp(_ a: CGFloat, _ b: CGFloat) -> CGFloat {
     return self < a ? a : (self > b ? b : self)
   }
 }
-extension TimeInterval {
-  internal func clamp(_ a: TimeInterval, _ b: TimeInterval) -> TimeInterval {
+
+internal extension TimeInterval {
+  func clamp(_ a: TimeInterval, _ b: TimeInterval) -> TimeInterval {
     return self < a ? a : (self > b ? b : self)
   }
 }
-extension CGPoint {
-  internal func translate(_ dx: CGFloat, dy: CGFloat) -> CGPoint {
+
+internal extension CGPoint {
```

Comment:
Technically this `internal` marking isn't required either,

> If you extend a public or internal type, any new type members you add have a default access level of internal. 

But since XCode keeps flipping how it feels about visibility I think it's not a bad idea to e explicit here.